include/spdlog/LICENSE

```diff
@@ -0,0 +1,25 @@
+The MIT License (MIT)
```

Comment:
Please scroll down all these `include/spdlog/*` files - this is the logging 3rd-party headers. Currently, I dropped them right into repo, but we might reconsider to make it as a submodule. In the next PR there will be one more dependency - headers only TOML parsing library - daemon configuration will be loaded/saved to `/etc/ocvsmd/ocvsmd.toml` file).

Reply:
Surely these `include`s need to be nested into a directory whose name makes it clear that these are third-party components. `3rdparty` sounds like a much better name.

Reply:
@pavel-kirienko  So, under `include/3rdparty/spdlog/*`, right?
@thirtytwobits do you agree?